src/features/account/AccountView.tsx

```diff
@@ -56,25 +60,6 @@ const AccountView = ({
     )
   }, [selectedBalance])
 
-  useEffect(() => {
-    if (accountNetType !== NetTypes.MAINNET) {
-      setFormattedHntPrice('Testnet')
-      return
-    }
-
-    if (!hntPrice && !selectedBalance) return
-
-    let price = hntPrice
-
-    if (selectedBalance) {
-      price = selectedBalance.hntPrice
-    }
-
-    CurrencyFormatter.format(price || 0, currency).then((p) =>
```

Comment:
Do we not need this anymore?

Reply:
Nope this is all happening in the TokenPricesTicker now